### ISSUE TITLE
released rbpdf 1.20.0 isn't built from the repository

### ISSUE THREAD

**Issue by ahorek** [naitoh/rbpdf#53]
```
gem install rbpdf
ERROR:  While executing gem ... (Errno::EINVAL)
    Invalid argument @ rb_sysopen - c:/mingw64/lib/ruby/gems/2.6.0/gems/rbpdf-1.20.0/test/json:
```

this looked suspicious to me, so I audited the gem. It doesn't contain any malicious code, but it was clearly not built against the github repository and there're some additional files, that breaks the installation.

for example
metadata contains
`- 'test/json:'`
test folder contains
`json_`
and there's no such file in the github repository

@naitoh could you fix the issue? imo, test files shouldn't be distributed at all. thanks

https://github.com/rubygems/rubygems/issues/2496

**Comment by naitoh** [comment]
Thank you for pointing out.

The reason is that garbage remained in my package creation environment.
Recreate this package.
Also remove the test resource from the gem.

**Comment by naitoh** [comment]
fixed af025c63035d768caeeeaa3f622e6a1ec0c0176f

**Comment by naitoh** [comment]
This problem has been fixed in rbpdf 1.20.1.

**Comment by ahorek** [comment]
great, thanks!
